OpenEdXMobile/res/layout/discussion_response_action_bar_layout.xml

```diff
@@ -72,4 +72,8 @@
             style="@style/discussion_responses_small_text"
             android:text="@string/discussion_responses_report_label" />
     </LinearLayout>
+
+    <View
+        style="@style/light_gray_separator"
+        android:layout_gravity="top" />
```

Comment:
I would still say it should be added to be proper. It being only 1 DIP height is an implementation detail, and you shouldn't rely on it to be small enough to not overlap. Another option would be to set a drawable that has a separator drawn at the top with the appropriate padding as the drawable of the container<sup>[1](#footnote1)</sup>. I'm not sure if such a drawable already exists or not.

----

<sup><a name="footnote1">1</a>: Note that if you create this drawable by drawing two overlapping `shape` drawables through a `layer-list`, this will have the side-effect of ensuring that the background will always be white. Lollipop added an attribute for `layer-list` items to set a specific size for themselves, but in the previous APIs you can only achieve this effect by using a nine-patch drawable.</sup>